BROWSER-SUPPORT.md

```diff
@@ -1,7 +1,5 @@
 # Browser support
 
-## Introduction
```

Comment:
I removed the extra heading as by definition the first content should introduce the thing.